### DISCUSSION TITLE
Bypass PD charging on TUF A15 2023 (FA507NU)

### DISCUSSION THREAD

**Discussion by emilka1337** [seerge/g-helper#2728] [Q&A]
Question is not mine. I found it on Reddit but I have the same problem. Does somebody know anything about it and is that possible to fix that behavior, maybe add bypass charging by software way?

> Hello, last week I bought A15 with 4050 It support PD with maximum power of 100W I've got lock on battery to keep it on 80% The problem is that when it's connected to PD it's discharging battery to 79% and charging to 80% with maximum power instead to take energy straight from power input. Example when laptop take ~50w - charger is going to 100w every 3-4 seconds always when battery go down to 79% and when battery level hit 80% it suddenly go down to 0w then again and again. I'm plenty sure it's going to destroy battery really fast. Am I wrong?

**Comment by seerge** [comment]
@emilka1337 hello, PD-bypass is controlled by firmware. You can't change that in the app anyhow.

As far as I know only new G14/G16 2024 support proper PD with bypass. And again only if device consumes less than 100w or so

**Comment by AgostinoA** [comment]
This problem also exists on G14 2025.
https://www.reddit.com/r/ZephyrusG14/comments/1k9qyui/2025_zephyrus_g14g16_does_not_support_usbc_bypass/
